docs/backend/auth-twitter.md

```diff
@@ -13,42 +13,43 @@ application.
 
 1. [Sign in](https://developer.twitter.com/apps) to your Twitter account.
 
-
-2. In order to get the API keys, you will have to
-   [apply for a developer account](https://developer.twitter.com/en/apply-for-access). Click the **Apply** button.
+2) In order to get the API keys, you will have to
+   [apply for a developer account](https://developer.twitter.com/en/apply-for-access).
+   Click the **Apply** button.
 
    ![twitter-up-1](https://user-images.githubusercontent.com/22895284/51078779-53139b00-16bb-11e9-911c-f232e229872a.png)
 
-3. Setup your Twitter account. Be sure you have your phone number and email
+3) Setup your Twitter account. Be sure you have your phone number and email
    address filled in.
 
    ![twitter-up-2](https://user-images.githubusercontent.com/22895284/51078780-53139b00-16bb-11e9-91d5-08c9365ff08f.png)
 
-4. Fill in your account information and give a name to your **developer
+4) Fill in your account information and give a name to your **developer
    account**.
 
    ![twitter-up-3](https://user-images.githubusercontent.com/22895284/51078781-53ac3180-16bb-11e9-8cf4-005efbb92d8a.png)
 
-5. Write down the reasons that you want to use Twitter API. Mention DEV's
+5) Write down the reasons that you want to use Twitter API. Mention DEV's
    community and describe the issues and tests and things that you want to work
    on. Copy it, you might use it later ;)
 
    ![twitter-up-4](https://user-images.githubusercontent.com/22895284/51078782-53ac3180-16bb-11e9-9937-c888ae40143c.png)
 
-6. Read :) and accept the Terms and Conditions.
+6) Read :) and accept the Terms and Conditions.
 
    ![twitter-up-5](https://user-images.githubusercontent.com/22895284/51078783-53ac3180-16bb-11e9-9cf1-8e009ada6e57.png)
 
-7. Verify your email address once more, and you will be done.
+7) Verify your email address once more, and you will be done.
 
-8. You are done.
+8) You are done.
```

Comment:
I don't have a preference, but I'm curious why these numbered lists were changed.

The biggest downside of doing this is that we'd probably want to make the same change everywhere else :slightly_frowning_face:

Reply:
I'm not really sure to be honest. Looks like some accidental change caused by my editor settings...? I'll check.

Reply:
Reverted that.